Makefile

```
@@ -12,18 +12,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-REGISTRY                            := eu.gcr.io/gardener-project/gardener
-APISERVER_IMAGE_REPOSITORY          := $(REGISTRY)/apiserver
-CONTROLLER_MANAGER_IMAGE_REPOSITORY := $(REGISTRY)/controller-manager
-SCHEDULER_IMAGE_REPOSITORY          := $(REGISTRY)/scheduler
-ADMISSION_IMAGE_REPOSITORY          := $(REGISTRY)/admission-controller
-SEED_ADMISSION_IMAGE_REPOSITORY     := $(REGISTRY)/seed-admission-controller
-GARDENLET_IMAGE_REPOSITORY          := $(REGISTRY)/gardenlet
-PUSH_LATEST_TAG                     := false
-VERSION                             := $(shell cat VERSION)
-EFFECTIVE_VERSION                   := $(VERSION)-$(shell git rev-parse HEAD)
-REPO_ROOT                           := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
-LOCAL_GARDEN_LABEL                  := local-garden
+REGISTRY                            	:= eu.gcr.io/gardener-project/gardener
+APISERVER_IMAGE_REPOSITORY          	:= $(REGISTRY)/apiserver
+CONTROLLER_MANAGER_IMAGE_REPOSITORY 	:= $(REGISTRY)/controller-manager
+SCHEDULER_IMAGE_REPOSITORY          	:= $(REGISTRY)/scheduler
+ADMISSION_IMAGE_REPOSITORY          	:= $(REGISTRY)/admission-controller
+SEED_ADMISSION_IMAGE_REPOSITORY     	:= $(REGISTRY)/seed-admission-controller
+GARDENLET_IMAGE_REPOSITORY          	:= $(REGISTRY)/gardenlet
+LANDSCAPER_GARDENLET_IMAGE_REPOSITORY   := $(REGISTRY)/landscaper-component-gardenlet
```

1. fix indentation
2. 
```suggestion
LANDSCAPER_GARDENLET_IMAGE_REPOSITORY   := $(REGISTRY)/landscaper-gardenlet
```